AdobeStockAsset/Test/Api/AssetRepository/SaveTest.php

```diff
@@ -137,7 +137,7 @@ private function saveAsset(array $data)
                 'operation' => self::SERVICE_NAME . self::SERVICE_OPERATION,
             ],
         ];
-        $requestData = ['asset' => $data];
+        $requestData = ['asset' => $data,];
```

Comment:
Such commas just do not look right. I think it's better to disable this rule in the IDE.